wsgi/odyssy/basic/templates/basic/index.html

```diff
@@ -26,8 +26,16 @@
                     <ul id="imp_announcement">
                         {% for announcement in imp_news_announcement %}
                             <li>
-                                <a class="alert-link" href="#">{{ announcement.title }}:</a>
-                                <div style="margin-left: 5px">{{ announcement.description }}</div>
+                            	{% if announcement.get_model_type == "Announcement" %}
```

Comment:
Instead of two if's we can use this here, https://docs.djangoproject.com/en/1.11/ref/templates/builtins/#firstof